hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestMORDataSourceStorage.scala

```diff
@@ -46,12 +46,12 @@ class TestMORDataSourceStorage extends SparkClientFunctionalTestHarness {
 
   @ParameterizedTest
   @CsvSource(Array(
-    "true,",
-    "true,fare.currency",
-    "false,",
-    "false,fare.currency"
+    "true,,false",
```

Comment:
The new partial update support is tested through `TestPartialUpdateForMergeInto`.  We don't have to change this test.  Did this test fail before this change?